src/anjaliS/PrimeNumbersAvgSum.java

```diff
@@ -0,0 +1,38 @@
+package anjaliS;
+
+import java.util.Scanner;
+
+public class PrimeNumbersAvgSum {
+	void PrimeAvgSum(int start, int end) {
+		int sum = 0, avg = 0, count = 0;
+		for (int index = start; index <= end; index++) {
+			boolean flag = false;
+			for (int j = 2; j <= index / 2; j++) {
+				if (index % j == 0)
+					flag = true;
+			}
+			if (!flag) {
+				System.out.print(index + " ");
+				count++;
+				sum += count;
+				avg = sum / count;
```

Comment:
once loop is end then you should perform avg formula, why we need to do operation for each iteration.